Vue/.gitignore

```diff
@@ -0,0 +1,62 @@
+# Logs
```

Comment:
Can we have just one .gitignore at the root of the repo?

Reply:
- [x] Done in 4d8a729

Reply:
we also need `.gitignore` per example, since one can fork the example and play around with it.

Reply:
I really don't think that's necessary, especially since you're throwing all sorts of other shit in here that isn't needed for the example at all

Reply:
FYI, It's the default Node template git provides.